internal/plugins/helm/v1/scaffolds/init.go

```diff
@@ -67,18 +66,13 @@ func (s *initScaffolder) newUniverse() *model.Universe {
 
 // Scaffold implements Scaffolder
 func (s *initScaffolder) Scaffold() error {
-	switch {
-	case s.config.IsV3():
-		if err := s.scaffold(); err != nil {
-			return err
-		}
-		if s.apiScaffolder != nil {
-			return s.apiScaffolder.Scaffold()
-		}
-		return nil
-	default:
-		return fmt.Errorf("unknown project version %v", s.config.Version)
```

Comment:
I thought that it was removed already.

Reply:
I thought so too. We did it in api but not init.

Reply:
OH .. good catcher, 👍